### PR TITLE
Reusable CICD for SciOps Workflows

### PR DESCRIPTION
**Pull request by drewyangdev**

### CODE REVIEW

File: .github/.staging_workflows/anchored_sciops_docker_images.yaml [hunk 1]
```diff
@@ -76,31 +73,41 @@ anchor-docker-login: &docker-login
...
           ## Get build info
-          cd ./docker
-          export REPO_NAME=$(echo ${{github.repository}} | awk -F\/ '{print $2}')
-          echo "::set-output name=repo_name::${REPO_NAME}"
+          cd ./main/docker/codebook_env
+          export REPO_NAME=${{github.event.repository.name}}
           export REPO_OWNER=${{github.repository_owner}}
 
           echo "${{secrets.RAW_DEPLOY_KEY}}" | base64 -di >> ${REPO_NAME}-deploy.pem
           chmod 400 ${REPO_NAME}-deploy.pem
```
Comment (guzman-raphael) [inline .github/.staging_workflows/anchored_sciops_docker_images.yaml]: A bit more verbose but think this is a bit easier for reviews.

```suggestion
          chmod u=r,g-rwx,o-rwx ${REPO_NAME}-deploy.pem
```

File: .github/.staging_workflows/anchored_sciops_docker_images.yaml [hunk 2]
```diff
@@ -118,161 +125,90 @@ jobs:
...
           export DEPLOY_KEY=${REPO_NAME}-deploy.pem
 
-          ## Build image
-          docker-compose -f docker-compose-standard_worker.yaml build
+          ## Build workers
+          find . -maxdepth 1 -type d \( -iname "*" ! -iname "*local*" \) | awk "NR>1" | cut -d/ -f2 | xargs -I {} bash -c \
```
Comment (guzman-raphael) [inline .github/.staging_workflows/anchored_sciops_docker_images.yaml]: Might make it a bit easier to work with if you use a heredoc. Here are some example references:
- https://linuxize.com/post/bash-heredoc/ (simple case)
- https://github.com/datajoint/miniconda3-docker/blob/fe9fb413af0a742ea5809b76ca9634cd00cf9d3e/tests/main.sh#L103-L107 (real world)

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@yambottle Great work! :construction_worker: 

Only adding minor points from our discussion.

File: .github/.staging_workflows/anchored_sciops_docker_images.yaml (hunk 2 above)
Comment (drewyangdev) [reply .github/.staging_workflows/anchored_sciops_docker_images.yaml]: xargs has issue working with heredoc... So I updated all xargs with while loop which is similar approach.

**Review by guzman-raphael — APPROVED** [review]
@yambottle Looking good! :sunglasses:

**Pull request merged**